src/socket.cpp

```diff
@@ -172,6 +232,15 @@ CSocket::~CSocket()
 
 void CSocket::SendPacket ( const CVector<uint8_t>& vecbySendBuf, const CHostAddress& HostAddr )
 {
+    union
```

Comment:
Can this go in a header somewhere as a type, so it can be re-used more easily?

Reply:
I guess, although that makes it more opaque when reading `socket.cpp`.

Reply:
You could argue that for every definition that isn't in line.